lib/bookmark_manager.rb

```diff
@@ -7,22 +7,27 @@ def initialize(user)
     @user = user
   end
 
-  def create(post_id:, name: nil, reminder_type: nil, reminder_at: nil, options: {})
-    post = Post.find_by(id: post_id)
+  def create(post_id:, topic_id: nil, name: nil, reminder_type: nil, reminder_at: nil, options: {})
+    if post_id == Bookmark::FOR_TOPIC_POST_ID
```

Comment:
I wonder if we just pass nil to post_id here? require topic_id / post_id here. 

from the JSON API we should simply omit sending the post_id. is there a away we can pull this off?

Reply:
Hmm I think you have a point. Let me try that way out, it will be clearer than a magic number.